generators/templates.go

```diff
@@ -810,7 +810,7 @@ func httpGet{{ initialCap .Name }}(w http.ResponseWriter, r *http.Request, vars
   defer collections.{{ .Name }}Mutex.Unlock()
 	obj := collections.{{ .Name }}s[key]
 	if obj == nil {
-		log.Errorf("{{ .Name }} %s not found", key)
+		log.Infof("{{ .Name }} %s not found", key)
 		return nil, errors.New("{{ .Name }} not found")
```

Comment:
The error returned could have the key as well (use fmt.Errorf). This can be done in a follow up PR